packages/frontend/editor-ui/src/composables/useImportCurlCommand.test.ts

```diff
@@ -292,5 +298,265 @@ describe('useImportCurlCommand', () => {
+			const parameters = toHttpNodeParameters(curl);
+			expect(parameters.url).toBe('https://reqbin.com/echo');
+			expect(parameters.method).toBe('PUT');
+			expect(parameters.sendBody).toBe(false);
+			expect(parameters.contentType).toBeUndefined();
+			expect(parameters.sendHeaders).toBe(false);
+			expect(parameters.sendQuery).toBe(false);
+		});
+
+		test('Should parse cURL command with custom header and multiple data fields', () => {
+			const curl =
+				'curl -X POST https://reqbin.com/echo -H "Authorization: Bearer token" -d "key1=value1" -d "key2=value2"';
+			const parameters = toHttpNodeParameters(curl);
+			expect(parameters.url).toBe('https://reqbin.com/echo');
+			expect(parameters.method).toBe('POST');
+			expect(parameters.sendBody).toBe(true);
+			expect(parameters.contentType).toEqual('form-urlencoded');
+			expect(parameters.bodyParameters?.parameters).toEqual([
+				{ name: 'key1', value: 'value1' },
+				{ name: 'key2', value: 'value2' },
+			]);
+			expect(parameters.sendHeaders).toBe(true);
+			expect(parameters.headerParameters?.parameters[0].name).toBe('Authorization');
+			expect(parameters.headerParameters?.parameters[0].value).toBe('Bearer token');
+		});
+
+		test('Should parse cURL command with custom header and binary data', () => {
+			const curl =
+				'curl -X POST https://reqbin.com/echo -H "Content-Type: application/octet-stream" --data-binary "@/path/to/file"';
+			const parameters = toHttpNodeParameters(curl);
+			expect(parameters.url).toBe('https://reqbin.com/echo');
+			expect(parameters.method).toBe('POST');
+			expect(parameters.sendBody).toBe(true);
+			expect(parameters.contentType).toBe('raw');
+			expect(parameters.rawContentType).toBe('application/octet-stream');
+			expect(parameters.sendHeaders).toBe(false);
+			expect(parameters.sendQuery).toBe(false);
+		});
+
+		test('Should parse cURL command with multiple headers and case-insensitive keys', () => {
+			const curl =
+				'curl -X POST https://reqbin.com/echo -H "content-type: application/json" -H "ACCEPT: application/json" -d \'{"key":"value"}\'';
+			const parameters = toHttpNodeParameters(curl);
+			expect(parameters.url).toBe('https://reqbin.com/echo');
+			expect(parameters.method).toBe('POST');
+			expect(parameters.sendBody).toBe(true);
+			expect(parameters.contentType).toBe('json');
+			expect(parameters.bodyParameters?.parameters[0].name).toBe('key');
+			expect(parameters.bodyParameters?.parameters[0].value).toBe('value');
+			expect(parameters.sendHeaders).toBe(true);
+			expect(parameters.headerParameters?.parameters[0].name).toBe('ACCEPT');
+			expect(parameters.headerParameters?.parameters[0].value).toBe('application/json');
+		});
+
+		test('Should parse cURL command with no URL', () => {
+			const curl = 'curl -X POST -d "key=value"';
+			expect(() => toHttpNodeParameters(curl)).toThrow('no URL specified!');
+		});
+
+		test('Should parse cURL command with invalid protocol', () => {
```

Comment:
[nits] This test case verifies a different function, so maybe should be in a separate describe() section